src/constants/common.js

```diff
@@ -66,4 +66,6 @@ module.exports = {
 	SORT_ORDER: 'sort_order',
 	URL: 'url',
 	FILE_PATH: 'filePath',
+	STATUS_RESOLVED: 'RESOLVED',
+	ALL_USER_ROLES: 'all',
```

Comment:
We have FILTER_ALL key also with the same value. Can we reuse / combine both? @priyanka-TL

Reply:
no let be separate

Reply:
ok